Spigot-API-Patches/0119-Add-an-asterisk-to-legacy-API-plugins.patch

```diff
@@ -16,12 +16,12 @@ index c0ff133de..b173e2ece 100644
      }
 +
 +    static boolean isLegacyPlugin(org.bukkit.plugin.Plugin plugin) {
```

Comment:
Had another peek, CB already has a method with a similar intent here, might be worth exposing that instead vs writing our own
```java
    public static boolean isLegacy(PluginDescriptionFile pdf) {
        return pdf.getAPIVersion() == null;
    }
```

Reply:
Different intent, am dum, ignore me on that one...